talk/basicconcepts/control.tex

```diff
@@ -1,15 +1,15 @@
-\subsection[Control]{Control instructions}
+\subsection[Control]{Control structures}
```

Comment:
I disliked the term control instructions, since they are not instructions but impose structures on the code. We could also call them `Control statements`, but control structures is probably the most used term.